move_base:
  ros__parameters:
    shutdown_costmaps: false

    controller_frequency: 5.0
    controller_patience: 5.0

    planner_frequency: 5.0
    planner_patience: 5.0

    oscillation_timeout: 0.0
    oscillation_distance: 0.5

    recovery_behavior_enabled: false #true
    clearing_rotation_allowed: true